### ISSUE TITLE
Improper handling of timestamp field during export to Avro.

### ISSUE THREAD

**Issue by wduczkowski** [datacontract/datacontract-cli#141]
Error during export occurs on the latest version of the Docker image datacontract/cli:main. I used 
as [orders-latest-nested](https://github.com/datacontract/datacontract-specification/blob/main/examples/orders-latest-nested/datacontract.yaml ) the source file, where the field "order_timestamp" is of type timestamp.

After executing the command:

`datacontractcli export --format avro datacontract.yaml --model orders
`
the tool exported the Avro schema in the following format:

```
{
  "type": "record",
  "name": "orders",
  "doc": "One record per order. Includes cancelled and deleted orders.",
  "fields": [
    {
      "name": "order_id",
      "doc": "An internal ID that identifies an order in the online shop.",
      "type": "string"
    },
    {
      "name": "order_timestamp",
      "doc": "The business timestamp in UTC when the order was successfully
registered in the source system and the payment was successful.",
      "type": "string"
    },
    {
      "name": "order_total",
      "doc": "Total amount the smallest monetary unit (e.g., cents).",
      "type": "long"
    },
    {
      "name": "customer_id",
      "doc": "Unique identifier for the customer.",
      "type": "string"
    },
    {
      "name": "customer_email_address",
      "doc": "The email address, as entered by the customer. The email address
was not verified.",
      "type": "string"
    },
    {
      "name": "address",
      "doc": "The delivery address of the customer.",
      "type": {
        "type": "record",
        "name": "address",
        "doc": "The delivery address of the customer.",
        "fields": [
          {
            "name": "street",
            "doc": "The street name and house number.",
            "type": "string"
          },
          {
            "name": "city",
            "doc": "The city name.",
            "type": "string"
          },
          {
            "name": "additional_lines",
            "doc": "Additional address lines, such as floor, apartment, or
company name.",
            "type": "array"
          }
        ]
      }
    },
    {
      "name": "processed_timestamp",
      "doc": "The timestamp when the record was processed by the data
platform.",
      "type": "string"
    }
  ]
}
```
In the above output, the field "order_timestamp" is of type String. The same situation applies to the field: processed_timestamp.

The timestamp should be converted to one of the logical Avro types specified here: https://avro.apache.org/docs/1.11.1/specification/#date

**Comment by jochenchrist** [comment]
I second this, we should support logical types (https://avro.apache.org/docs/1.11.1/specification/#logical-types)

`date` could be exported to
```
"type": "int"
"logicalType": "date"
```

However, timestamps are not simple, as Avro looses the time zone information.

We clearly can support `timestamp_ntz`, it could be exported to
```
"type": "long"
"logicalType": "local-timestamp-millis"
```

What happens if we have a timestamp with timezone (`timestamp` or `timestamp_tz` alias)? What would you expect here?

**Comment by wduczkowski** [comment]
The `timestamp` should be in UTC, while `timestamp_tz` could take the system default or be obtained from a parameter passed in the CLI.

**Comment by jochenchrist** [comment]
In the Data Contract Specification, `timestamp` is an alias for `timestamp_tz`.

When exporting to AVRO, we could export both to, and assume that those are in UTC.
```
"type": "long"
"logicalType": "timestamp-millis"
```

**Comment by jochenchrist** [comment]
Resolved with #199
